coursedata/level-defaults/nb_NO.yaml

```diff
@@ -1,354 +1,354 @@
-    start_code: '{print} hello world!'
+    intro_text: "## Velkommen til Hedy\nPå Nivå 1 kan du bruke kommandoene `{print}`, `{ask}` og `{echo}`.\nSkriv koden i programmeringsfeltet, eller trykk på den grønne boksen i eksempelkodeblokken og koden vil bli skrevet for deg!\nPrøv koden self med den grønne 'Kjør kode' knappen under programmeringsfeltet.\n\nDu kan skrive tekst til skjermen ved å bruke `{print}` kommandoen.\n"
+    example_code: "## Eksempelkode\n```\n{print} Hallo!\n{print} Velkomen til Hedy!\n```\n"
+    intro_text_2: "Du kan også spørre etter inndata med `{ask}` og gjenta det tilbake med en `{echo}`kommando.\n"
+    intro_text_3: "La oss starte! Ikke sikker på hva du vil lage? I de neste fanene finner du ideer til programmer å bygge.\n"
+    example_code_2: "## Eksempelkode\n```\n{ask} Hva er navnet ditt?\n{echo} hallo\n```\n"
+    start_code: '{print} hallo verden!'
     commands:
     -   name: '{print}'
-        explanation: '{print} something with `{print}`.'
-        example: 'Example: {print} Hello welcome to Hedy!'
-        demo_code: '{print} Hello welcome to Hedy!'
+        explanation: '{print} noe med `{print}`.'
+        example: 'Eksempel: {print} Hallo, velkommen til Hedy!'
+        demo_code: '{print} Hallo velkommen til Hedy!'
     -   name: '{ask}'
-        explanation: ask something with `{ask}`.
-        example: 'Example: {ask} What is your favorite color?'
-        demo_code: '{ask} What is your favorite color?'
+        explanation: Still et spørsmål med `{ask}`.
+        example: 'Eksempel: {ask} Hva er favorittfargen din?'
+        demo_code: '{ask} Hva er favorittfargen din?'
     -   name: '{echo}'
-        explanation: repeat something using `{echo}`.
-        example: 'Example: {echo} so your favorite color is '
-        demo_code: "{ask} What is your favorite color?\n{echo} so your favorite color is"
-    -   name: '{print} emojis'
-        explanation: print an emoji with `{print}`
-        example: 'Example: {print} 🙋‍ 🌍 ❗'
+        explanation: gjenta noe med å bruke `{echo}`.
+        example: 'Eksempel: {echo} så favorittfargen din er '
+        demo_code: "{ask} Hva er favorittfargen din?\n{echo} så favorittfargen din er"
+    -   name: '{print} emojier'
+        explanation: print en emoji med `{print}`
+        example: 'Eksempel: {print} 🙋‍ 🌍 ❗'
         demo_code: '{print} 🙋 🌍 ❗'
     -   name: '{forward}'
-        explanation: Draw a line with `{forward}`.
-        example: 'Example: {forward} 100'
+        explanation: Tegn en linje med `{forward}`.
+        example: 'Eksempel: {forward} 100'
         demo_code: '{forward} 100'
     -   name: '{turn}'
-        explanation: turn the drawing turtle with `{turn}`
-        example: 'Example: {turn}'
-        demo_code: "{forward} 25\n{turn} left\n{forward} 25\n{turn} right"
+        explanation: snu tegneskilpadden med `{turn}`
+        example: 'Eksempel: {turn}'
+        demo_code: "{forward} 25\n{turn} venstre\n{forward} 25\n{turn} høyre"
```

Comment:
This doesn't work as the keywords are not (yet) translated in this PR. When creating the placeholder structure `left` and `right` where not yet localized. From now on they should be between curly brackets as well. Long story short:

Translated keywords should **never** be placed directly in code. Always use placeholders or the English keywords to make sure the code is always valid.

Reply:
Fixed in weblate!